### DISCUSSION TITLE
Why few members in the core team are fully inactive?

### DISCUSSION THREAD

**Discussion by rohit2sharma95** [twbs/bootstrap#31146] [General] [answered]
There are few members in the core team who have not committed in the repo since a long time 🤭

**Comment by wnunezc** [comment]
interesting question: put a photo ...

**Comment by patrickhlauke** [comment]
and your point/suggestion is?

**Comment by XhmikosR** [comment]
Yeah, we need to clear this up at some point because it gives the false image that we have many team members while we do not.

But anyway, that's mostly an internal thing :)

**Reply by wnunezc** [reply]
purge inactive members ... and recruit news ... they need new blood ... and see blood run ...
if a team and its leaders are made, Maintaining an internal and external image that there are opportunities to grow and that who does not want to participate is not essential.

**Reply by patrickhlauke** [reply]
"yeah, how dare those people that at one point contributed a lot of their spare time and energy but are now taking more of a backseat for whatever reason still bask in the glory of being listed as team members"

**Reply by wnunezc** [reply]
> "yeah, how dare those people that at one point contributed a lot of their spare time and energy but are now taking more of a backseat for whatever reason still bask in the glory of being listed as team members"

Give him a certificate of veteram army and go...

**Comment by mdo** [comment, marked as answer]
I'll be taking a pass at updating this in our second alpha to clarify some things :). Suffice to say folks are free to come and go, contribute a little or a lot, and to do so at any point in time.
